applet/src/md380-2.032.c

```diff
@@ -12,8 +12,21 @@
 
 //Firmware calls to 2.032.
 
-
```

Comment:
what do you think about the idea to rename all firmware functions with the header md380_ to iritationen with avoiding newlib?